src/main/charts/bamboo/templates/service-monitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.monitoring.serviceMonitor.create }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "common.names.fullname" . }}-service-monitor
+  labels:
+  {{- with .Values.monitoring.serviceMonitor.prometheusLabelSelector }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  endpoints:
+  - interval: {{ printf "%ds" (default 30 .Values.monitoring.serviceMonitor.scrapeIntervalSeconds)}}
```

Comment:
Can we move the default into the `values.yaml` file to make it more visible?

Reply:
I'd leave it here just in case someone runs helm upgrade with --reuse-values and the default values.yaml will be ignored.